quickwit/quickwit-metastore/src/metastore/mod.rs

```diff
@@ -141,6 +141,19 @@ pub trait Metastore: Send + Sync + 'static {
         split_metadata: SplitMetadata,
     ) -> MetastoreResult<()>;
 
+    /// Stages multiple splits.
+    async fn stage_splits(
```

Comment:
This is not atomic. Remove. Let's remove `stage_split` as well.

Reply:
After discussing with @fulmicoton, we realized that we'd like this operation to be nilpotent. Can you update the doc string of this method and implement the corresponding behavior?